sup.rb

```diff
@@ -0,0 +1,41 @@
+require "language/go"
```

Comment:
I'm not really sure if we should keep them in this repository or just rely on homebew and a package maintainer. 

However, my opinion is that we should keep this file in `release/sup.rb` as a source of truth. And later on we may have some `make release_brew` that automatically checkout and push a branch on the dev account(like pressly/homebrew). Then the dev simply create a PR on homebrew using github UI. 
 
It also help power user who may run: `brew install https://raw.githubusercontent.com/pressly/sup/master/release/sup.rb` if homebrew doesn't merge quickly for whatever reason.